### PR TITLE
Procfile: delay proxy waiting for initial cluster

### LINKED ISSUE #3647: proxy in `goreman start` needs a long time to find endpoint list
https://gist.github.com/philips/7d58b20399baecfeeaa0

When `goreman start`, proxy fails to get endpoint list at the first time, and it may wait 30s to give it another try, which leads to the result that proxy may have 30s that cannot redirect correctly.

This should be improved.

/cc @philips

### PR DESCRIPTION
**Pull request by gyuho**
This fixes #3647 by delaying proxy start by 3 seconds. Without this, proxy
starts at the same time as initial cluster and since the default proxy director
refresh interval is 30-second, if cluster is not ready at first trial, the
proxy misses to discover them and has to wait another 30-seconds, which delays
the proxying for first 30-second.

### CODE REVIEW

**Comment by gyuho** [comment]
@yichengq As you suggested, I put 1 second but it wasn't enough. For me, 3 seconds work well, that proxy does not miss discovering the initial cluster.

Thanks!

File: Procfile [hunk 1]
```diff
@@ -2,4 +2,5 @@
...
 etcd2: bin/etcd -name infra2 -listen-client-urls http://127.0.0.1:22379 -advertise-client-urls http://127.0.0.1:22379 -listen-peer-urls http://127.0.0.1:22380 -initial-advertise-peer-urls http://127.0.0.1:22380 -initial-cluster-token etcd-cluster-1 -initial-cluster 'infra1=http://127.0.0.1:12380,infra2=http://127.0.0.1:22380,infra3=http://127.0.0.1:32380' -initial-cluster-state new
 etcd3: bin/etcd -name infra3 -listen-client-urls http://127.0.0.1:32379 -advertise-client-urls http://127.0.0.1:32379 -listen-peer-urls http://127.0.0.1:32380 -initial-advertise-peer-urls http://127.0.0.1:32380 -initial-cluster-token etcd-cluster-1 -initial-cluster 'infra1=http://127.0.0.1:12380,infra2=http://127.0.0.1:22380,infra3=http://127.0.0.1:32380' -initial-cluster-state new
-proxy: bin/etcd -name infra-proxy1 -proxy=on -listen-client-urls http://127.0.0.1:2379 -initial-cluster 'infra1=http://127.0.0.1:12380,infra2=http://127.0.0.1:22380,infra3=http://127.0.0.1:32380'
+# wait a few seconds until initial cluster is ready to be proxied.
```
Comment (xiang90) [inline Procfile]: Let's be accurate? wait 3 seconds?

**Comment by xiang90** [comment]
LGTM

**Comment by gyuho** [comment]
Updated. Thanks for review @xiang90.

**Comment by yichengq** [comment]
LGTM

**Comment by philips** [comment]
Can we please just fix the proxy?

**Comment by xiang90** [comment]
@philips One way I can see it to let proxy aggressively sync until find a useable cluster then slow down. Maybe worth a try. /cc @gyuho @yichengq

**Comment by gyuho** [comment]
@xiang90 Makes sense. I can try that in another PR. Thanks!

**Pull request merged**